RELEASES.md

```diff
@@ -90,43 +88,54 @@ whichever is longer. Additionally, releases may have an extended security suppor
-| [1.0](https://github.com/containerd/containerd/releases/tag/v1.0.3)  | End of Life   | December 5, 2017   | December 5, 2018                                        |
-| [1.1](https://github.com/containerd/containerd/releases/tag/v1.1.8)  | End of Life   | April 23, 2018     | October 23, 2019                                        |
-| [1.2](https://github.com/containerd/containerd/releases/tag/v1.2.13) | End of Life   | October 24, 2018   | October 15, 2020                                        |
-| [1.3](https://github.com/containerd/containerd/releases/tag/v1.3.10) | End of Life   | September 26, 2019 | March 4, 2021                                           |
-| [1.4](https://github.com/containerd/containerd/releases/tag/v1.4.13) | End of Life   | August 17, 2020    | March 3, 2022                                           |
-| [1.5](https://github.com/containerd/containerd/releases/tag/v1.5.18) | End of Life   | May 3, 2021        | February 28, 2023                                       |
-| [1.6](https://github.com/containerd/containerd/releases/tag/v1.6.36) | LTS           | February 15, 2022  | next LTS + 6 months                                     |
-| [1.7](https://github.com/containerd/containerd/releases/tag/v1.7.23) | Active        | March 10, 2023     | active(May 5, 2025), extended(EOL of 1.6)               |
-| [2.0](https://github.com/containerd/containerd/releases/tag/v2.0.0)  | Active        | November 5, 2024   | max(November 5, 2025 or release of 2.1 + 6 months)      |
-| [2.1](https://github.com/containerd/containerd/milestone/48)         | Next          | TBD                | TBD                                                     |
-
-> **_NOTE_** containerd v1.7 will end of life at the same time as v1.6 LTS. Due to
-> [Minimal Version Selection](https://go.dev/ref/mod#minimal-version-selection) used
-> by Go modules, 1.7 must be supported until EOL of all 1.x releases. Once 1.7 is in
-> extended support, it will continue to accept security patches in addition to client
-> changes relevant for package importers using the 1.6 LTS daemon.
+Long term stable (_LTS_) releases are sponsored by at least two maintainers for at least two
+years after their initial _minor_ release. The maintainers of the _LTS_ branch may commit to a
+longer period or extend the support period as needed. These branches will accept bug reports and
+backports until the end of life date. They may also accept a wider range of patches than non-_LTS_
+releases to support the longer term maintainability of the branch, including library dependency,
+toolchain (including Go) and other version updates which are needed to ensure each release is built
+with fully supported dependencies. Feature backports are up to the discretion of the maintainers who
+own the branch but should be rejected by default. There is no limit to the number of _LTS_ branches
```

Comment:
Probably a soft-limit can be set.
It doesn't sound realistic to keep maintaining more than 3 LTS branches

Reply:
Updated to say "no defined limit", we can define a limit if needed in the future